lib/resin-lint.coffee

```diff
@@ -7,8 +7,10 @@ merge = require('merge')
 optimist = require('optimist')
 coffeelint = require('coffeelint')
 reporter = require('coffeelint/lib/reporters/default')
+tslint = require('tslint')
 
-CONFIG_PATH = path.join(__dirname, '../config/coffeelint.json')
+CS_CONFIG_PATH = path.join(__dirname, '../config/coffeelint.json')
+TS_CONFIG_PATH = path.join(__dirname, '../config/tslint.json')
```

Comment:
Just an idea: How about using an object, containing the config filename (& possibly the target file types as well)?